### DISCUSSION TITLE
Search internal content fields

### DISCUSSION THREAD

**Discussion by niknikita** [OrchardCMS/OrchardCore#15858] [Q&A]
Hi!
How can I set up a search on the Manage Content page so that I can search not only by headings but also by internal fields? for example htmlbody

![image](https://user-images.githubusercontent.com/20708134/79673690-c03aa800-81fd-11ea-8362-dd348436c973.png)

**Comment by hishamco** [comment]
Almost - if not all - the filter bar in the admin site is searched in the heading to avoid any server hit again

**Comment by sebastienros** [comment]
https://docs.orchardcore.net/en/dev/docs/guides/implement-fulltext-search/#seventh-step-fine-tune-full-text-search
